pages/mesosphere/dcos/services/data-science-engine/1.0.1/dynamic-allocation/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: layout.pug
+navigationTitle: Dynamic Allocation 
+excerpt: Enabling Dynamic Allocation with Shuffle Service for DC/OS Data Science Engine Spark
+title: Dynamic Allocation
+menuWeight: 7
+enterprise: true
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+---
+
+# Prerequisites
```

Comment:
Let's add a sentence about what dynamic allocation is before the prerequisites. We can copy it from Spark docs and provide a link

Reply:
Copied this from https://spark.apache.org/docs/latest/job-scheduling.html#dynamic-resource-allocation:

"Spark provides a mechanism to dynamically adjust the resources your application occupies based on the workload. This means that your application may give resources back to the cluster if they are no longer used and request them again later when there is demand. This feature is particularly useful if multiple applications share resources in your cluster."

I put it before "Prerequisites". What do you think?

Reply:
Sounds good! Thanks, Sarah.